src/Chapter 2/RemoveDups2.py

```diff
@@ -0,0 +1,89 @@
+"""2.1 Remove Dupes"""
+
+class Node(object):
+    """Node Class"""
+    def __init__(self, d, n=None):
+        self.data = d
+        self.next_node = n
+
+    @property
+    def next(self):
```

Comment:
Avoid having propeties refer to anything but a directly related private variable

```
@property
def next_node(self):
    return self._next_node
```